studygroups/views/api.py

```diff
@@ -225,8 +226,9 @@ def get(self, request):
                 .annotate(last_meeting_date=Subquery(active_meetings.reverse().values('meeting_date')[:1]))\
                 .filter(last_meeting_date__lt=today)
 
-        q = request.GET.get('q', None)
-        if q:
+        q = request.GET.get('q', '').strip()
+
+        if len(q) > 0:
```

Comment:
@s-kennedy `if q` is equivalent to `len(q) > 0` See https://docs.python.org/3.8/library/stdtypes.html#truth-value-testing